Core/include/eASTDetectorConstruction.hh

```diff
@@ -37,6 +38,7 @@ class eASTDetectorConstruction : public G4VUserDetectorConstruction
   private:
     eASTDetectorConstructionMessenger* messenger;
     G4VPhysicalVolume* fWorld = nullptr;
+    eASTMagneticField* fField = nullptr;
```

Comment:
One improvement could have this pointer be const. Detector construction does not need to modify the field. I didn't try this out. It would require that the Activate function called from ConstructSDandField become const too, but that should be no problem until we have a need to keep the equation or chord finder pointer around. Can't see that happening, I think.